content.js

```diff
@@ -72,51 +72,42 @@ var popupInitialised = false;
 
 //should skips happen at all
 var disableSkipping = false;
-chrome.storage.sync.get(["disableSkipping"], function(result) {
-    let disableSkippingStorage = result.disableSkipping;
-    if (disableSkippingStorage != undefined) {
-        disableSkipping = disableSkippingStorage;
-    }
-});
+let disableSkippingStorage = SB.config.disableSkipping;
+if (disableSkippingStorage != undefined) {
+    disableSkipping = disableSkippingStorage;
+}
 
 //should skips be manual
 var disableAutoSkip = false;
-chrome.storage.sync.get(["disableAutoSkip"], function(result) {
-    let disableAutoSkipStorage = result.disableAutoSkip;
-    if (disableAutoSkipStorage != undefined) {
-        disableAutoSkip = disableAutoSkipStorage;
-    }
-});
+let disableAutoSkipStorage = SB.config.disableAutoSkip;
+if (disableAutoSkipStorage != undefined) {
+    disableAutoSkip = disableAutoSkipStorage;
+}
 
 //should view counts be tracked
 var trackViewCount = false;
-chrome.storage.sync.get(["trackViewCount"], function(result) {
-    let trackViewCountStorage = result.trackViewCount;
-    if (trackViewCountStorage != undefined) {
-        trackViewCount = trackViewCountStorage;
-    } else {
-        trackViewCount = true;
-    }
-});
+let trackViewCountStorage = SB.config.trackViewCount;
+if (trackViewCountStorage != undefined) {
+    trackViewCount = trackViewCountStorage;
+} else {
+    trackViewCount = true;
+}
 
 //if the notice should not be shown
 //happens when the user click's the "Don't show notice again" button
 //option renamed when new notice was made
 var dontShowNotice = false;
-chrome.storage.sync.get(["dontShowNotice"], function(result) {
-    let dontShowNoticeAgain = result.dontShowNotice;
-    if (dontShowNoticeAgain != undefined) {
-        dontShowNotice = dontShowNoticeAgain;
-    }
-});
+let dontShowNoticeAgain = SB.config.dontShowNotice;
+if (dontShowNoticeAgain != undefined) {
+    dontShowNotice = dontShowNoticeAgain;
+}
+	
 //load the legacy option to hide the notice
 var dontShowNoticeOld = false;
-chrome.storage.sync.get(["dontShowNoticeAgain"], function(result) {
-    let dontShowNoticeAgain = result.dontShowNoticeAgain;
-    if (dontShowNoticeAgain != undefined) {
-        dontShowNoticeOld = dontShowNoticeAgain;
-    }
-});
+let dontShowNoticeAgain = SB.config.dontShowNoticeAgain;
```

Comment:
This will cause an error from multiple definitions.

Reply:
Outdated